packages/api/src/app.ts

```diff
@@ -197,6 +197,14 @@ export class ApiApp {
               newestBlock: params[2],
             });
             break;
+          case CustomRPCMethods.skandha_config:
+            result = await skandhaApi.getConfig();
+            // skip hexlify for this particular rpc
+            return res.status(200).send({
```

Comment:
Can we pls avoid hardcoding and move the http status codes and messages to a common constants file?

Reply:
done